docs/deploy-for-ocr.md

```diff
@@ -3,48 +3,102 @@
 ```
 git clone https://github.com/SuSy-One/susy-chainlink-auditor
 cd susy-chainlink-auditor/
-mkdir chainlink_data
-cd chainlink_data
+mkdir chainlink-1
+cd chainlink-1
 ```
 ## Access settings
 
 Please note that Chainlink node enforces a password policy that requires a password to have at least 3 numbers and 3 symbols.
-First, create two config files in `./chainlink_data`:
+First, create two config files in folder `./chainlink-1`:
 
-`.password` -- node access password
+`.password` -- node access password (of your choosing):
 ```
 echo "MyVeryStrongP@ssword123" > .password
 ```
 
-`.api` - login/password for Chainlink Operator GUI access
+`.api` -- login/password for Chainlink Operator GUI access (both of your choosing, used only locally):
+```
+touch .api // Crating the file
+nano .api // Opening editor
+```
+Add these two lines in the editor:
 ```
 myemail@email.com
 StringPasswordForApi123
 ```
+Press ctrl+X to exit, when prompt asks - save the changes.
 
 ## Node launch
-Edit `chainlink.env` and provide node launch parameters. 
+Makse sure `chainlink.env` has correct launch parameters - currently there is no need to change anything, repo contains all the setup parameters we need. 
+
+If you are using brand new server - install latest `Docker-compose`, there might be issues with setup using apt-get, better use curl as per [instructions here](https://docs.docker.com/compose/install/).  
 
 Launch the node:
 ```
 docker-compose up -d
 ```
 
-Use http://localhost:7788 to access Chainlink Operator GUI (or use ssh tunneling if hosting on a remote machine).
+Check the status of the container using command `docker ps`. There should be several containers containers:
+```
+CONTAINER ID   IMAGE                                    COMMAND                  CREATED          STATUS                    PORTS                                       NAMES
+a46ba5a1f652   susy-chainlink-auditor_node1             "chainlink local nod…"   10 minutes ago   Up 10 minutes (healthy)   0.0.0.0:7788->6688/tcp, :::7788->6688/tcp
+2261359f83bd   postgres:13.1-alpine                     "docker-entrypoint.s…"   10 minutes ago   Up 10 minutes             0.0.0.0:5433->5432/tcp, :::5433->5432/tcp
+3545acdd1fe0   susy-chainlink-auditor_bridge-peg-usd    "/initiator pegUsd"      10 minutes ago   Up 10 minutes                                           
+1d487355a08b   susy-chainlink-auditor_bridge-peg-base   "/initiator pegBase"     10 minutes ago   Up 10 minutes
+```
+If the status isn't **Up** - but instead something like **Restarting (1)** - it means that there are some configuration problems, you can check the logs: 
+```
+docker logs susy-chainlink-auditor_node1_1 // enter the name of your failing image here
+```
+Most likely the issue is with the credentials folder - make sure the naming is correct (as seen above) and the files `.password` and `.api` exist.
```

Comment:
Since the folder name above is correct, this could be due to a completely different issue 
I would omit this